README.md

```diff
@@ -82,6 +81,60 @@ ReactDOM.render(<App />, appRoot);
 
 ## Options
 
+#### `element` : PropTypes.string or React.PropTypes.func (default: "input")
+
+You can specify element="textarea". For Example:
+
+```js
+<DebounceInput
+  element="textarea"
+/>
+```
+
+Will result in
+
+```js
+<textarea />
+```
+
+Note: when rendering a `<textarea />` you may wish to set `forceNotifyByEnter = { false }` so the user can make new lines without forcing notification of the current value.
+
+This package has only been tested with `<input />` and `<textarea />` but should work with any element which has `value` and `onChange` props.
+
+You can also use a custom react component as the element. For Example:
+
+```js
+<DebounceInput
```

Comment:
Wow, that is a great side-effect! Basically fixes https://github.com/nkbt/react-debounce-input/issues/5